src/User.cpp

```diff
@@ -543,18 +543,19 @@ CString& CUser::ExpandString(const CString& sStr, CString& sRet) const {
 	CString sTime = CUtils::CTime(time(nullptr), m_sTimezone);
 
 	sRet = sStr;
-	sRet.Replace("%user%", GetUserName());
-	sRet.Replace("%defnick%", GetNick());
-	sRet.Replace("%nick%", GetNick());
 	sRet.Replace("%altnick%", GetAltNick());
+	sRet.Replace("%bindhost%", GetBindHost());
+	sRet.Replace("%defnick%", GetNick());
+	sRet.Replace("%empty%", "");
```

Comment:
I think %empty% should be in the very end, to make it possible to write `%ni%empty%ck%` as a workaround to put a non-expanded `%nick%`. If such usage is to be supported at all...